docs/source/raft_configuration.md

```diff
@@ -330,5 +330,34 @@ channels (this does not mean they share data --- they do not!), if a Raft node i
 part of the consenter set in many channels, you might want to lengthen the amount
 of time it takes to trigger an election to avoid inadvertent leader elections.
 
+2. As aforementioned, write-ahead logs (wal) and snapshots should be persisted in
+order to attain safety property of Raft protocol. However, it may happen if a node
```

Comment:
Is the abbreviation wal or WAL? It's used both ways in this section.

Reply:
changed all to be WAL